proposals/78:upgrade-17/.gitignore

```diff
@@ -0,0 +1,6 @@
+# Generated Artifacts to Ignore in CI
+add-LEMONS/
+add-OLIVES/
+upgrade-bank/
+upgrade-provisionPool/
+upgrade-orch-core/
```

Comment:
I think these are generated in the agoric-sdk context but in this context, they're checked in.